system-contract-dapp-playground/src/fonts/styreneA-webfont/1802-UFGULA.css

```diff
@@ -0,0 +1,240 @@
+/*
+        This font software is the property of Commercial Type.
+
+        You may not modify the font software, use it on another website, or install it on a computer.
+
+        License information is available at http://commercialtype.com/eula
+        For more information please visit Commercial Type at http://commercialtype.com or email us at info[at]commercialtype.com
+
+        Copyright (C) 2018 Schwartzco Inc.
+        License: 1802-UFGULA     
```

Comment:
Erm you might have to remove this. We only want to use open source software. It doesn't sounds like this font package allows
Given the warning I'm not sure how it intends to be used.
What package or tool is this from?

Reply:
Hello @Nana-EC, thank you for the feedback. I noticed that the Styrene A font is not listed on Google Fonts, and I learned that Hedera employs Styrene A as the primary sans-serif font family. That's why I prefer to stick to this font. These files were downloaded from the Hedera branding site which can be found here https://hederabrandcentral.frontify.com/d/Tmocz52AXpLj/brand-assets#/brand-assets/typography/download-fonts